packages/decoupled-data-wordpress/src/lib/remove-domain.ts

```diff
@@ -0,0 +1,15 @@
+import { replace } from 'lodash';
```

Comment:
please check if those very generic utilities are used elsewhere - possibly it makes sense to either create a separate utils-package or keep them in decoupled and expose them from there.

Reply:
Yeah, but in any case, this is the same result, as long as both decoupled and any other package use lodash on the same version

Reply:
no, it's not about lodash, but the `removeDomain` utility itself

Reply:
LOL yeah, got it! Yeah, I moved the utils temporarely. I agree I will check if they are used anywhere else but I like the idea of `decoupled-utils-package`